src/main/java/mods/eln/misc/BasicContainer.java

```diff
@@ -178,7 +178,7 @@ protected boolean mergeItemStack(ItemStack par1ItemStack, int par2, int par3, bo
         return flag1;
     }
 
-    @Override
+
     public ItemStack slotClick(int arg0, int arg1, int arg2, EntityPlayer arg3) {
```

Comment:
If this is no longer overriding anything, then whatever it was trying to do won't work and you need to either fix it or add a TODO.